### ISSUE TITLE
[BUG/ISSUE] Tagged CO simulation in GC v13.3.2

### ISSUE THREAD

**Issue by elutsch** [geoschem/geos-chem#1131]
### What institution are you from?
University of Toronto

### Description of the problem

There is an apparent unit error for the production of CO from CH4 and NMVOC oxidation terms. These terms (ProdCOfromCH4 and ProdCOfromNMVOC, respectively) are archived from the ProdLoss menu of the HISTORY.rc of a full chemistry simulation in units of **molec/cm3/s**. However, HEMCO assumes these are unitless which results in unphysically large CO concentrations produced from the tagged CO simulation.

### Description of troubleshooting performed
In the HEMCO_Config.rc file, the Prod/Loss terms are specified as: 
```
#==============================================================================
# --- Quantities needed for CO chemistry ---
#==============================================================================
(((GLOBAL_OH
* GLOBAL_OH      ../geosfp_2x2.5_fullchem/OutputDir/GEOSChem.SpeciesConc.$YYYY$MM$DD_0000z.nc4     SpeciesConc_OH    2010-2019/1-12/1/0 C xyz 1     * - 1 1
)))GLOBAL_OH
(((PROD_CO_CH4
* PCO_CH4       ../geosfp_2x2.5_fullchem/OutputDir/GEOSChem.ProdLoss.$YYYY$MM$DD_0000z.nc4         ProdCOfromCH4     2010-2019/1-12/1/0 C xyz 1     * - 1 1
)))PROD_CO_CH4
(((PROD_CO_NMVOC
* PCO_NMVOC      ../geosfp_2x2.5_fullchem/OutputDir/GEOSChem.ProdLoss.$YYYY$MM$DD_0000z.nc4        ProdCOfromNMVOC   2010-2019/1-12/1/0 C xyz 1     * - 1 1
)))PROD_CO_NMVOC
```
In the `/GeosCore/tagged_co_mod.F90,` a unit conversion is applied on line 499:
```
          ! HEMCO brings in PCO in kg/m3, so we need to also
          ! apply a conversion to molec/cm3/s
!~           PCO_CH4_MCM3S = PCO_CH4(I,J,L) * kgm3_to_mcm3sCO * FAC_DIURNAL
```

Removing the unit conversion factor of `kgm3_to_mcm3sCO` fixes the issue, with the same fix applied to the NMVOC production term on line 512. With this correction, the CO concentrations match the full-chem concentrations quite well as expected.

**Comment by yantosca** [comment]
Thanks for writing @elutsch.  The tagCO simulation has lagged behind other GEOS-Chem development.  We can make this fix into 13.4.1, as it is a simple update.

**Comment by msulprizio** [comment]
This issue has been addressed by pull request #1151 and will be included in GEOS-Chem 13.4.0.
